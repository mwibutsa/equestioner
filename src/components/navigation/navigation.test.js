import '../../setupTest';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import React from 'react';
import Navigation from '../index';

describe('<Navigation />', () => {
  test('it should render app correctly', () => {
    const wrapper = shallow(<Navigation></Navigation>);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
