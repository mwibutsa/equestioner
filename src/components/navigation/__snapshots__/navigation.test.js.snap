// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navigation /> it should render app correctly 1`] = `
<div>
  Navigation
</div>
`;
